### ISSUE TITLE
Present tense for pluar

### ISSUE THREAD

**Issue by maxonfjvipon** [volodya-lombrozo/jtcop#248]
What if we test something in the plural? For example users, then our tests may be written differently but still in present tense:
- it has -> they have
- it contains -> they contain
- and so on...

`jtcop` swears at it `because the test name has to be written using present tense`

**Comment by volodya-lombrozo** [comment]
@maxonfjvipon Yes, you are right. Actually it is the significant problem which we have in the `jtcop` for now. To be honest we check only the `s` letter of the first word in the method name :)
The proper solution would be to use ML techniques to solve that problem. I started the implementation of that feature by using [Pos Tagger](https://en.wikipedia.org/wiki/Part-of-speech_tagging):

- https://github.com/volodya-lombrozo/jtcop/issues/101
- https://github.com/volodya-lombrozo/jtcop/tree/main/src/main/java/com/github/lombrozo/testnames/rules/ml

I've faced with some technical problems like a ML model size and the output of this model isn't precise. So, I'm working on it an if you have a desire to help, please join :)

**Comment by h1alexbel** [comment]
@volodya-lombrozo assign me, please

**Comment by volodya-lombrozo** [comment]
@h1alexbel Sure, please, go ahead

**Comment by 0pdd** [comment]
@maxonfjvipon the puzzle [#433](https://github.com/volodya-lombrozo/jtcop/issues/433) is still not solved.
